packages/playground/blueprints/src/lib/steps/login.ts

```diff
@@ -21,44 +19,24 @@ export type LoginStep = {
 	 */
 	username?: string;
 	/**
-	 * The password to log in with. Defaults to 'password'.
+	 * @deprecated The password field is deprecated and will be removed in a future version.
+	 * Only the username field is required for user authentication.
 	 */
 	password?: string;
 };
 
 /**
  * Logs in to Playground.
- * Under the hood, this function submits the [`wp-login.php`](https://developer.wordpress.org/reference/files/wp-login.php/) [form](https://developer.wordpress.org/reference/functions/wp_login_form/)
- * just like a user would.
+ * Under the hood, this function sets the `PLAYGROUND_AUTO_LOGIN_AS_USER` constant.
+ * The `auto_login.php` mu-plugin uses that constant to log in the user on the first load.
+ * The plugin is located in and loaded automatically by the `@wp-playground/wordpress` package.
```

Comment:
```suggestion
 * This step depends on the `@wp-playground/wordpress` package because
 * the plugin is located in and loaded automatically by the `@wp-playground/wordpress` package.
```